src/cmd-push-container-manifest

```diff
@@ -4,18 +4,24 @@
 # arguments provided by the user.
 
 import argparse
+import json
 import os
 import sys
 from cosalib.container_manifest import create_and_push_container_manifest
 from cosalib.builds import Builds
 from cosalib.meta import GenericBuildMeta
+from cosalib.cmdlib import runcmd
 from cosalib.cmdlib import sha256sum_file
 
 sys.path.insert(0, os.path.dirname(os.path.abspath(__file__)))
 
 
 def main():
     args = parse_args()
+    map_arch = {}
+    map_arch['arm64'] = 'aarch64'
+    map_arch['amd64'] = 'x86_64'
```

Comment:
Aside: we should make a function for converting between these arch names in `cmdlib` like we have in golang. But not necessary to do in this PR!